### ISSUE TITLE
Will someone received when comment is made?

### ISSUE THREAD

**Issue by watsonl** [DataShades/ckanext-comments#3]
Will author or system admin received comment?

**Comment by mutantsan** [comment]
No. But you can implement it using [signals](https://github.com/DataShades/ckanext-comments/blob/master/ckanext/comments/signals.py).
